subprojects/signing/src/integTest/groovy/org/gradle/plugins/signing/SigningSamplesSpec.groovy

```diff
@@ -60,6 +60,18 @@ class SigningSamplesSpec extends AbstractIntegrationSpec {
         module.assertArtifactsPublished("maven-metadata.xml", "conditional-${module.publishArtifactVersion}.pom", "conditional-${module.publishArtifactVersion}.jar")
     }
 
+    @UsesSample('signing/gnupg-signatory')
+    def "use gnupg signatory"() {
```

Comment:
This test might fail on our CI infrastructure if the `gpg` executable is not installed. We might have to put in additional work into the infrastructure to ensure the right setup for tests.